robottelo/ui/base.py

```diff
@@ -415,6 +415,14 @@ def edit_entity(self, edit_loc, edit_text_loc, entity_value, save_loc):
         self.text_field_update(edit_text_loc, entity_value)
         self.click(save_loc)
 
+    def set_limit(self, limit):
+        """Specify content host limit value for host collection or activation
+        key entities.
+        """
+        self.click(locators['hostcollection.usage_limit_checkbox'])
+        if limit != 'Unlimited':
+            self.field_update('hostcollection.usage_limit', limit)
+
```

Comment:
Actually, we have 
`check_all_values` - used only for `location` entity.
`auto_complete_search` is used only for two specific entities
`remove_parameter` used only for two specific entities (even docstring states that)

all of them have even less reasons to be put into `base` module